packages/cohort_banking/src/model/requests.rs

```diff
@@ -6,7 +6,8 @@ pub struct CandidateData {
     pub readset: Vec<String>,
     pub writeset: Vec<String>,
     pub statemap: Option<Vec<HashMap<String, Value>>>,
-    // The "snapshot" is intentionally messing here. We will compute it ourselves before feeding this data to Talos
+    // The "snapshot" is intentionally missing here. We will compute it ourselves before feeding this data to Talos
+    pub on_commit: Option<Value>,
```

Comment:
Please add this to readme as well;

https://github.com/kindredgroup/talos/blob/master/cohort_sdk_client/README.md#about-jscertificationcandidate